bokeh/core/enums.py

```diff
@@ -345,7 +351,8 @@ def enumeration(*values, **kwargs):
 LineJoin = enumeration("miter", "round", "bevel")
 
 #: Specify a location in plot layouts
-Location = enumeration("above", "below", "left", "right")
+_Location = Literal["above", "below", "left", "right"]
```

Comment:
Since these are intended to be imported elsewhere it would be better to give them "public" names, e.g `LocationType`, `SizingModeType`